### PR TITLE
Model-1: Random Forest & XGBoost

### PR DESCRIPTION
**Pull request by PanwarJayant**
This PR updates the main repos model section with the following functionalities for Model-1:

- Random forest classifier
- XGBoost classifier

### CODE REVIEW

File: models/model1_part2.ipynb [hunk 1]
```diff
@@ -0,0 +1,611 @@
+{
```
Comment (gaursagar21) [inline models/model1_part2.ipynb]: rename this file; its not clear what model1_part2 is

Comment (gaursagar21) [inline models/model1_part2.ipynb]: <p>Keep the imports together.</p><p>pandas has already been imported earlier.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (gaursagar21) [inline models/model1_part2.ipynb]: <pre class="ql-syntax" spellcheck="false">Line #2.    df = pd.read_csv("/content/drive/MyDrive/markov/Initial_Dataset.csv")
</pre><p>Duplicate of block 2</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (gaursagar21) [inline models/model1_part2.ipynb]: <p>If this code will be reused in multiple notebooks, I recommend moving it to a <code>utils/text_preprocessing.py</code> file.</p><p>Same code is in the other PR </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (gaursagar21) [inline models/model1_part2.ipynb]: <pre class="ql-syntax" spellcheck="false">Line #1.    # SEPARATING THE DATA CLASSES
</pre><p>nit. Give the block a heading, don't use all caps comments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (gaursagar21) [inline models/model1_part2.ipynb]: <pre class="ql-syntax" spellcheck="false">Line #2.    # only 1000 additional samples, this is done to address imbalanced classes
</pre><p>This comment isn't clear. What are you trying to do here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (PanwarJayant) [reply models/model1_part2.ipynb]: We are thinking of moving both the notebooks of the first model into a single one. We are not sure if we will use the same code for the coming models but if we do, we will make sure to include the redundant code in a utils folder like you have suggested.

**Pull request merged**